stable_baselines3/common/preprocessing.py

```diff
@@ -111,5 +120,9 @@ def get_action_dim(action_space: spaces.Space) -> int:
     elif isinstance(action_space, spaces.Discrete):
         # Action is an int
         return 1
+    elif isinstance(action_space, spaces.MultiDiscrete):
+        return int(len(action_space.shape))
```

Comment:
it won't work as `action_space.shape` is not defined here :/
(and same for MultiBinary)